app/src/main/kotlin/com/wire/android/util/FileUtil.kt

```diff
@@ -78,6 +84,21 @@ suspend fun Uri.toByteArray(context: Context, dispatcher: DispatcherProvider = D
     }
 }
 
+suspend fun Uri.toDrawable(context: Context, dispatcher: DispatcherProvider = DefaultDispatcherProvider()): Drawable? {
+    val dataUri = this
```

Comment:
what is the purpose of this 🤔

Reply:
just to increase code readability.

Reply:
ok, I think you can see it as quick as without it as it is an extension functon on Uri, but up to you I am fine with both approaches 🆗